### ISSUE TITLE
`fillStyle` as gradient

### ISSUE THREAD

**Issue by cipriancaba** [reactivemarkets/react-financial-charts#518]
#### I'm submitting a...

- [ ] bug
- [x] feature
- [ ] chore

#### What is the current behavior
Can only submit solid colors

#### What is the expected behavior
Ability to pass a gradient. The current docs states that you can pass a gradient
```
/**
     * Color, gradient, or pattern to use for fill.
     */
    readonly fillStyle?: string;
```

#### Please tell us about your environment

- Version: 1.1.0
- Browser: [all]

#### Other information
I am trying to replicate the `createVerticalLinearGradient ` from `react-stockcharts` but can't seem to figure out how. Here is the stockcharts variant but not sure how this should work in react-financial-charts https://codesandbox.io/s/github/rrag/react-stockcharts-examples2/tree/master/examples/AreaChart?file=/src/Chart.js:387-415

**Comment by markmcdowell** [comment]
Hi yes, I was originally allowing to pass in canvas gradients but it was removed as you needed to have a reference to an internal canvas to create the gradient. In react-stockcharts it was only supported by svg and we're trying to move everything to canvas.

So in short can't be done yet, but I do want to add support for it.

**Comment by cipriancaba** [comment]
Are there any plans to expose the canvas instance or maybe you are considering an encoded gradient type in the fillStyle property that might allow for the creation of the gradient?

**Comment by markmcdowell** [comment]
Released in [v1.3.0](https://github.com/reactivemarkets/react-financial-charts/releases/tag/v1.3.0)
